package.json

```diff
@@ -20,8 +20,8 @@
     "test": "cake test"
   },
   "dependencies": {
-    "make-sync": "~0.2.0",
-    "fibers": "~1.0.1",
+    "make-sync": "git://github.com/dnice-asana/node-make-sync.git#master",
```

Comment:
I think this should be pointing to github.com/Asana, right?

Reply:
yes!